package.json

```diff
@@ -47,10 +47,11 @@
     "@babel/core": "7.24.7",
     "@babel/preset-env": "7.24.7",
     "@types/jest": "29.5.12",
-    "@types/node": "20.14.5",
+    "@types/node": "20.14.15",
```

Comment:
Please include the caret to support newer minor versions in the future (need to update the lock file as well by running `yarn`).